explorer/GrapherGrammar.ts

```diff
@@ -181,9 +182,20 @@ export const GrapherGrammar: Grammar = {
         })),
     },
     sortColumnSlug: {
-        ...EnumCellDef,
+        ...SlugDeclarationCellDef,
         keyword: "sortColumnSlug",
         description:
             "This setting is only respected when `sortBy` is set to `column`",
     },
+    stackMode: {
+        ...EnumCellDef,
+        keyword: "stackMode",
+        description:
+            "Show chart in absolute (default) or relative mode. Only works for some chart types.",
+        terminalOptions: Object.keys(StackMode).map((keyword) => ({
```

Comment:
We strive to keep them identical, but `Object.values` would be more correct here right?

Reply:
you're absolutely right, good catch!